File: BomberEngine/Core/Assets/Types/Scheme.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using BombermanCommon.Resources.Scheme;

namespace BomberEngine.Core.Assets.Types
{
    public class Scheme : Asset
    {
        private SchemeInfo info;

        public Scheme(SchemeInfo info)
        {
            this.info = info;
        }

        protected override void OnDispose()
        {   
        }
    }
}

File: Utils/Builder/src/bc/assets/types/Scheme.java
package bc.assets.types;

import java.io.File;

import bc.assets.Asset;
import bc.assets.AssetInfo;

public class Scheme extends Asset
{
	private static AssetInfo info; // = new AssetInfo("Scheme", "sch", "SchemeImporter");
	
	public Scheme()
	{
		super(info);
	}
	
	public Scheme(String name, File file)
	{
		super(info, name, file);
	}

}
